src/_includes/_layouts/Home.jsx

```diff
@@ -113,6 +113,31 @@ export default ({ comp, title }) => {
             redirect_url={"projects"}
           />
         </section>
+        <section className="h-screen flex flex-col items-center justify-center py-16 gap-y-20 max-lg:h-fit ">
```

Comment:
The padding between the `Projects` and `How We Do It` sections and the padding between`How We Do It` and sponsors are not the same. Can you modify so that the padding between the sections mentioned above is the same

Reply:
This hasn't been fixed